migration/v0.20.x-v0.21.x/upgrade-apps.md

```diff
@@ -0,0 +1,7 @@
+# Upgrade v0.20.x to v0.21.x
+
+## Prerequisites
+
+## Steps
+
+1. Remove grafana-ops chart with all the dashboards: `migration/v0.20.x-v0.21.x/remove-grafana-ops.sh`
```

Comment:
With this I'm guessing that you need to run `bin/ck8s apply sc` afterwards?

Reply:
yes, i added the apply sc|wc step